core/src/main/java/com/graphhopper/util/Parameters.java

```diff
@@ -191,9 +191,9 @@
      */
     public static final class DETAILS {
 
-        private static final String DETAILS_PREFIX = "details.";
```

Comment:
and the dot is missing, so yes, a revert is best: DETAILS_PREFIX="details." or are you using this elsewhere where the dot is bad?